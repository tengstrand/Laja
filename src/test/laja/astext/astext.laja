#write "target/test-output/astext.out"
  #set (a = 123)
  #set (begin = "#>")
  #set (end = "<#")
#> #set (b = <#{a})
{begin}some text{end}
b = {b}
#end
