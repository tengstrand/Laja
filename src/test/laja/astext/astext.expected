 #set (b = 123)
#>some text<#
b = NULL
